documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,150 @@
+      2. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
+   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+7. Events
+   1. Each event must have a planned number of rounds.
+      1. Each round of each event must have a planned format and time limit.
+      2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
+      3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
+   2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
+      1. Other events may only be held unofficially.
+   3. If the main event of the competition is not the 3x3x3 Cube, it should be publicly mentioned in the competition's information.
+8. Schedule
+   1. The schedule must be published on the WCA website using the WCA schedule tool.
+   2. The schedule should be manageable by the competition's WCA Delegate(s) and adhere to the WCA quality standards.
+9. Championships
+   1. A championship type is assigned to competitions that are National, Continental, or World Championships.
+      1. A competition must be located in the region where the championship type is selected.
+      2. A competition may have multiple championship types.
+      3. A championship type for a region must only be assigned to one competition in a year.
+      4. An additional selection process is conducted by the WCA Board for organizing Continental and World Championships.
+   2. Naming
+      1. Continental and World Championships names must begin with "WCA".
+      2. If used, the word "Championship" must be singular in the competition name.
+      3. Competitions which are not National, Continental, or World championships should avoid names that are similar to championship names.
+   3. Date(s)
+      1. National Championships should not be held on the same date(s) as Continental or World championships.
+      2. Continental Championships must not be held on the same date(s) as World Championships.
+   4. Championships should have all the officially recognized WCA events.
+   5. Championships must recognize and award champion titles and their respective prizes to the winners in the region.
+   6. Organization teams of National Championships should involve all the WCA Delegates of the respective countries, at the discretion of the Senior Delegate.
+10. Optional requirements
+    1. Require competitors to submit puzzles at a specific time before an event is scheduled to begin.
+    2. Prohibit competitors from participating in specific combinations of events.
```

Comment:
10.1 and 10.2 should be full sentences (like every other paragraph before).

Reply:
I think 10.1 and 10.2 should actually be nested underneath 10.3 and 10.1.1 and 10.1.2. Anyway, I've suggested as complete sentences.